### ISSUE TITLE
Is dbz84244-mysql5-v3.sql.bz2 available elsewhere?

### ISSUE THREAD

**Issue by mogglemoss** [andimiller/wormhol.es#4]
http://zofu.no-ip.de/dbz84244/dbz84244-mysql5-v3.sql.bz2 doesn't resolve.

**Comment by Sylnai** [comment]
I do not believe that database is actually required, I can't see anything obviously from it in my actual db
